### ISSUE TITLE
Array.remove() should mutate the existing array and return the mutated array

### ISSUE THREAD

**Issue by El-Dringo-Brannde** [El-Dringo-Brannde/Perfected-Prototypes#28]
Currently, 

the only way to mutate the array is to re-reference itself such as..
``` javascript
let arr = [1,2,3,4]


arr = arr.remove(2)
console.log(arr)
// => [1,3,4]
```
A change should be made such that it still returns the new value of the array, but also mutates the original array. 

For example, 

``` javascript
let arr = [1,2,3,4]


arr.remove(2)
console.log(arr)
// => [1,3,4]
```

**Comment by decarbonite** [comment]
May I work on this issue? :)

**Comment by El-Dringo-Brannde** [comment]
@decarbonite feel free my friend, along with anything else you may want to work on in the project

**Comment by decarbonite** [comment]
Thank you.
I have two approaches in mind for this issue, using _.remove from lodash but i have to name it something else so it doesnt crash the program cuz it has the same name as the main function. Or using splice to mutate the array. Which one would more suitable?

**Comment by El-Dringo-Brannde** [comment]
I'd like splice personally, it avoids importing another section of lodash code which inflates the package size. As opposed to `slice` which is I think can be a lot smaller, I'd need to look at the lodash `remove` function though

**Comment by El-Dringo-Brannde** [comment]
Closed with #34
